modules/php/States/ResolveActionTrait.php

```diff
@@ -95,7 +94,11 @@ private function _action_resolveAction($args)
     self::setGameStateValue("actionToResolve", -1);
 
     $game = Utils::getGame();
-    $game->checkWinConditions();
+
+    // If we have a forced move, we cannot win yet
```

Comment:
I am not entirely sure this will always be true.
But since "Let's do that again" and "Use what you take" only impact the discard pile or another hand, it doesn't really make a difference just yet anyway, right?

Reply:
Well, the trick at the moment is to add the card in your hand before it's automatically played. But if this extra card means you now have 10 and that's the goal, it could mean you win when you actuary shouldn't.